eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/EventSenderOptions.java

```diff
@@ -30,18 +33,31 @@ public EventSenderOptions partitionId(String partitionId) {
      * @param retry The retry policy used to govern retry attempts when an issue is encountered while sending.
      * @return The updated SenderOptions object.
      */
-    public EventSenderOptions retry(RetryPolicy retry) {
+    public EventSenderOptions retry(Retry retry) {
         this.retry = retry;
         return this;
     }
 
+    /**
+     * Sets the default timeout to apply when sending events. If the timeout is reached, before the Event Hub
+     * acknowledges receipt of the event data being sent, the attempt will be considered failed and considered to be
```

Comment:
"will be considered failed and considered to be retried". Do you really mean that the retry will only be considered, or do you mean it is considered failed and **will** be retried?